core/src/main/java/hudson/model/Run.java

```diff
@@ -1469,6 +1479,14 @@ public String toString() {
     	return new ByteArrayInputStream(charset != null ? message.getBytes(charset) : message.getBytes());
     }
    
+    /**
+     * Returns a Reader that reads from the log file.
```

Comment:
```suggestion
     * Reads from the log file.
```

Stating the return type of a method (and without `@code` or `@link`, at that) is not helpful.